shadowsocks/cipher_testing.go

```diff
@@ -22,21 +22,25 @@ import (
 	"github.com/shadowsocks/go-shadowsocks2/shadowaead"
 )
 
+const testCipher = "chacha20-ietf-poly1305"
+
 // MakeTestCiphers creates a CipherList containing `numCiphers` fresh AEAD ciphers.
```

Comment:
Update the comment to mention it returns the generated secrets.

Reply:
I followed your other suggestion so this no longer applies.